### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by teoguso**

### CODE REVIEW

**Comment by malteserteresa** [comment]
Ok but we will need to remove the retrieve_data_from_s3_bucket.py as that depends on boto3

**Review by malteserteresa — COMMENTED** [review]
Aren't there boto3 dependencies in the code?

File: requirements.txt [hunk 1]
```diff
@@ -2,10 +2,8 @@
 -e .
 
 # external requirements
-awscli
-boto3==1.9.250
-click
-coverage
+awscli==1.16.300
+coverage==4.5.4
```
Comment (malteserteresa) [inline requirements.txt]: Is this going to work as we have a dependency to boto3?

**Review by malteserteresa — COMMENTED** [review]
Can you also remove the s3 script?

File: requirements.txt (hunk 1 above)
Comment (teoguso) [reply requirements.txt]: I don't know. I'm looking at circelCI now and there's another issue: if pip install issues errors (wanrings?) the build will succeed anyway.

Comment (teoguso) [reply requirements.txt]: We also have stuff like this in the build now ,which I don't understand:
```
astroid 2.3.3 has requirement typed-ast<1.5,>=1.4.0; implementation_name == "cpython" and python_version < "3.8", but you'll have typed-ast 1.3.5 which is incompatible.
tensorboard 1.14.0 has requirement setuptools>=41.0.0, but you'll have setuptools 40.8.0 which is incompatible.
```

**Comment by teoguso** [comment]
It's really confusing. It all runs fine now and I don't know exactly why. I guess this is it. Can you review it and merge @malteserteresa ?
I included the aws dependency with DVC (it wasn't before) so that should have been taken care of.

**Comment by malteserteresa** [comment]
I'll check it tomorrow!

**Review by malteserteresa — DISMISSED** [review]
There is a file called retrieve_data_from_s3_bucket.py that needs to be removed before this is ok to merge

**Review by malteserteresa — APPROVED** [review]

**Pull request merged**